docs/sources/reference/commandline/cli.md

```diff
@@ -1440,9 +1440,7 @@ The `docker rename` command allows the container to be renamed to a different na
 
       -a, --all=false       Show all containers (default shows just running)
       --before=""           Show only container created before Id or Name.
-      -f, --filter=[]       Provide filter values. Valid filters:
-                              exited=<int> - containers with exit code of <int>
-                              status=(restarting|running|paused|exited)
+      -f, --filter=[]       Provide filter values.
```

Comment:
Perhaps it would be clearer (and consistent) to write what it *does*, not what you *provide*, e.g. "Filter containers (by conditions)". Not a great text, perhaps you can come up with something better.